specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/examples/Diagnostics_GetHostingEnvironmentDetectorResponse.json

```diff
@@ -0,0 +1,136 @@
+                                        "0",
+                                        "0"
+                                    ],
+                                    [
+                                        "2018-03-27T00:35:00Z",
+                                        "781627",
+                                        "0",
+                                        "1",
+                                        "0"
+                                    ],
+                                    [
+                                        "2018-03-27T00:40:00Z",
+                                        "785017",
+                                        "0",
+                                        "0",
+                                        "0"
+                                    ],
+                                    [
+                                        "2018-03-27T00:45:00Z",
+                                        "783518",
+                                        "0",
+                                        "0",
+                                        "0"
+                                    ],
+                                    [
+                                        "2018-03-27T00:50:00Z",
+                                        "785783",
+                                        "0",
+                                        "0",
+                                        "0"
+                                    ],
+                                    [
+                                        "2018-03-27T00:55:00Z",
+                                        "772874",
+                                        "0",
+                                        "0",
+                                        "0"
+                                    ],
+                                    [
+                                        "2018-03-27T01:00:00Z",
+                                        "787162",
+                                        "0",
+                                        "0",
+                                        "0"
+                                    ],
+                                    [
+                                        "2018-03-27T01:05:00Z",
+                                        "782036",
+                                        "0",
+                                        "0",
+                                        "0"
+                                    ],
+                                    [
+                                        "2018-03-27T01:10:00Z",
+                                        "784642",
+                                        "0",
+                                        "0",
+                                        "0"
+                                    ]
+                                ]
+                            },
+                            "renderingProperties": {
+                                "renderingType": 2,
```

Comment:
This is defined as an enum but this is an integer. Same elsewhere.

Reply:
I have changed it to return a string, which is the actual way that we return enums.